### PR TITLE
Kathryn ods patch 1

### PR DESCRIPTION
**Pull request by kathryn-ods**
# Overview

**What does this pull request do?**
updates example data - componentRecords fields were listing statementIds, now they list recordIds

**How can a reviewer test or examine your changes?**

**Who is best placed to review it?**

Closes #

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. [See the Handbook](https://openownership.github.io/bods-dev-handbook/translations.html)

## Documentation & Release

- [ ] I've checked that the documentation builds without failing. [See the Handbook](https://github.com/openownership/data-standard#building-the-documentation-locally)
- [ ] I've thought about how and when this needs to be released. [See the Handbook](https://openownership.github.io/bods-dev-handbook/standard_releases.html)      
- [ ] I've updated the changelog, if necessary. [See the Handbook](https://openownership.github.io/bods-dev-handbook/standard_releases.html#changelog)
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. [See the Handbook](https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md)

### CODE REVIEW

**Comment by kd-ods** [comment]
@kathryn-ods - you can remove your changes to examples/bods-package-fi-soe.json from this PR, since I fixed the component recordId in PR #727

**Comment by kathryn-ods** [comment]
> @kathryn-ods - you can remove your changes to examples/bods-package-fi-soe.json from this PR, since I fixed the component recordId in PR #727

Have done this now and re-requested a review

**Comment by kathryn-ods** [comment]
having an issue with the link check tests - https://www.iso20022.org/market-identifier-codes showing as broken

**Comment by kathryn-ods** [comment]
ok I removed and readded the link and that seems to have worked

**Comment by kd-ods** [comment]
@kathryn-ods - I'll take a look at this Wed.

**Review by kd-ods — CHANGES_REQUESTED** [review]
A couple of the example files could do with checking again. 

It's a headache getting these componentRecords lists right.

File: examples/mutilple-indirect-ownership-2.json [hunk 1]
```diff
@@ -300,12 +300,12 @@
         }
       ],
       "componentRecords": [
-        "59368bde-e38d-4a95-9a6c-8ef6aec54f9f",
-        "829ec2d1-6ecb-4ca6-874f-c9c59f2cd843",
-        "426507cf-e5ac-45c4-b6b0-92952a954623",
-        "3a1ba916-5f3d-4335-860d-3f7c20283c42",
-        "a56a0ced-447e-420d-9a07-42dcbc960a5f",
-        "337d8ba1-9d0d-4a64-bc5d-820c95a127fc"
+        "1e049760d6c7",
```
[the remark this answers is not in the document]
Comment (kathryn-ods) [reply examples/mutilple-indirect-ownership-2.json]: Thanks Kadie - I mapped them out in Miro to confirm these and your edits are right  https://miro.com/app/board/uXjVLfqBOP8=/?share_link_id=907051498258

**Comment by kathryn-ods** [comment]
@kd-ods I think this is out of date now? These changes should have been made in the other PR

**Pull request closed without merging**